docker-compose.yml

```diff
@@ -30,6 +46,8 @@ services:
       - PORT=80
       - NOREFRESH=1
       - LISTEN=srly-ose-nginx
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
```

Comment:
Why do we need this?

Reply:
Ahh yes... It's necessary for the viewer service to access the ZeroMQ PUB socket at port 10001.
Since `srly-ose-wifi-connect` is in host networking mode, the viewer will not be able to refer to the `srly-ose-wifi-connect` service by its service name.

Alternatively, we can just use the default host gateway IP. I can't exactly remember, but it's something like `172.x.x.x`. That way, we can omit the changes.